vue/components/ui/molecules/scrollable-items/scrollable-items.vue

```diff
@@ -0,0 +1,260 @@
+                iconStrokeWidth: 3,
+                ...this.buttonIconProps
+            };
+        },
+        showArrows() {
+            const hideArrows = this.autoHideArrows && !(this.scrollableWidth > 0);
+            return this.arrows && !hideArrows;
+        },
+        selectedIndex() {
+            return this.items.findIndex(item => item.value === this.selectedData);
+        },
+        isButtonPreviousDisabled() {
+            return this.selectedIndex <= 0;
+        },
+        isButtonNextDisabled() {
+            return this.selectedIndex >= this.items.length - 1;
+        }
+    },
+    watch: {
+        selected(value) {
+            this.selectedData = value;
+        },
+        selectedData(value) {
+            this.snapSelectedToCenter();
+            this.$emit("update:selected", value);
+        }
+    },
+    created: function() {
+        window.addEventListener("resize", this.onResize);
+        window.addEventListener("mouseup", this.onMouseUp);
+        window.addEventListener("mousemove", this.onMouseMove);
+    },
+    destroyed: function() {
+        window.removeEventListener("mousemove", this.onMouseMove);
+        window.removeEventListener("mouseup", this.onMouseUp);
+        window.removeEventListener("resize", this.onResize);
+    },
+    methods: {
+        itemClasses(item, index) {
+            const base = {};
+            base[`item-${item.value}`] = true;
+            if (item.value === this.selectedData) base.selected = true;
+            if (this.items.length && index === this.items.length - 1) base.last = true;
+            return base;
+        },
+        calculateScrollableWidth() {
+            if (!this.$refs["items-container"]) return;
+            this.scrollableWidth = this.$refs["items-container"].scrollLeftMax;
+        },
+        snapSelectedToCenter() {
+            this.$refs["items-container"].childNodes[this.selectedIndex].scrollIntoView({
+                behavior: "smooth",
+                inline: "center"
+            });
+        },
+        onResize(event) {
+            this.calculateScrollableWidth();
+        },
+        onButtonIconPreviousClick(event) {
+            this.selectedData = this.items[this.selectedIndex - 1].value;
+        },
+        onItemsContainerWheel(event) {
+            this.$refs["items-container"].scrollLeft += event.deltaY * -this.scrollSpeed;
+        },
+        onItemsContainerMouseDown(event) {
+            this.isMouseDown = true;
+        },
+        onMouseUp(event) {
+            this.isDraggingItems = this.isMouseDown = false;
+        },
+        onMouseMove(event) {
+            if (!this.isMouseDown) return;
+            this.isDraggingItems = true;
+
+            this.$refs["items-container"].scrollLeft -= event.movementX;
+        },
+        onItemMouseUp(event, item) {
+            if (this.isDraggingItems) return;
+
+            this.selectedData = item.value;
+            this.$emit("button-click", event, item);
+        },

```

Comment:
Mobile devices are already able to do this without any code. This is what in meant in the ticket by `Scrolling can be done through dragging (--> given by the browser <--) or by clicking the side arrows`

It's ok not to have this in non-touchable devices, it's typical. Besides that, in touchable devices the implementation will never be as good as the way they do it is obviously not using the javascript engine.

Reply:
So should I remove this feature or keep it?

Reply:
Please remove it, less code = happy us